### PR TITLE
Fix case of LOGGER and typos

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Changed the case of the `logger` variable in  `private static final Logger logger` to all upper case to match all other uses. 

Also stumbled upon a few spelling/grammar mistakes and took the occasion to fix them.

### CODE REVIEW

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/interceptor/CmmnCommandInvoker.java [hunk 1]
```diff
@@ -76,9 +76,7 @@ protected void executeOperation(CommandContext commandContext, boolean isStoreCa
         if (runnable instanceof CmmnOperation) {
             CmmnOperation operation = (CmmnOperation) runnable;
 
-            if (logger.isDebugEnabled()) {
```
Comment (filiphr) [inline modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/interceptor/CmmnCommandInvoker.java:79]: Let's leave the if check here. When doing a call to a method with varArgs an array is instantiate on every call. This is really bad for performance and memory efficiency. Doing an if check on the other side is much better. 

This places is a hot path that would be affected by the unnecessary varArgs array creation.

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/CmmnParserImpl.java [hunk 2]
```diff
@@ -37,7 +37,7 @@
  */
 public class CmmnParserImpl implements CmmnParser {
 
-    private final Logger logger = LoggerFactory.getLogger(CmmnParserImpl.class);
+    private final Logger LOGGER = LoggerFactory.getLogger(CmmnParserImpl.class);
```
Comment (filiphr) [inline modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/CmmnParserImpl.java:40]: This is an instance field, not a `static final` one. The logger should stay lowercase here.

File: modules/flowable-dmn-engine/src/main/java/org/flowable/dmn/engine/impl/interceptor/DmnCommandInvoker.java [hunk 3]
```diff
@@ -56,9 +56,7 @@ protected void executeOperations(CommandContext commandContext, boolean isStoreC
         DmnEngineAgenda agenda = CommandContextUtil.getAgenda(commandContext);
         while (!agenda.isEmpty()) {
             Runnable runnable = agenda.getNextOperation();
-            if (logger.isDebugEnabled()) {
-                logger.debug("Executing agenda operation {}", runnable);
-            }
+            LOGGER.debug("Executing agenda operation {}", runnable);
```
Comment (filiphr) [inline modules/flowable-dmn-engine/src/main/java/org/flowable/dmn/engine/impl/interceptor/DmnCommandInvoker.java:59]: Same comment as for the `CmmnCommandInvoker`

**Review by filiphr — CHANGES_REQUESTED** [review]
Thanks for the PR @dbmalkovsky. I've left some comments regarding some of the loggers.

File: modules/flowable-dmn-engine/src/main/java/org/flowable/dmn/engine/impl/interceptor/DmnCommandInvoker.java (hunk 3 above)
Comment (dbmalkovsky) [reply modules/flowable-dmn-engine/src/main/java/org/flowable/dmn/engine/impl/interceptor/DmnCommandInvoker.java]: Done

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/CmmnParserImpl.java (hunk 2 above)
Comment (dbmalkovsky) [reply modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/parser/CmmnParserImpl.java]: Done

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/interceptor/CmmnCommandInvoker.java (hunk 1 above)
Comment (dbmalkovsky) [reply modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/impl/interceptor/CmmnCommandInvoker.java:79]: Done

**Comment by dbmalkovsky** [comment]
The JDK 13 failure is an issue with fetching dependencies.

**Pull request merged**